deploy/charts/rawfile-csi/values.yaml

```
@@ -1,27 +1,136 @@
+# -- Name of the registered CSI Driver in the cluster
 provisionerName: "rawfile.csi.openebs.io"
 
-defaults: &defaults
-  image:
-    repository: docker.io/openebs/rawfile-localpv
-    tag: 0.8.0
-    pullPolicy: Always
-  resources:
-    limits:
-      cpu: 1
-      memory: 100Mi
-    requests:
-      cpu: 10m
-      memory: 100Mi
+global:
+  # -- Default image registry for Images from DockerHub
+  imageRegistry: docker.io
+  # -- Default image registry for Images from Kubernetes (registry.k8s.io)
+  k8sImageRegistry: registry.k8s.io
+  # -- Default pull policy for images
+  imagePullPolicy: IfNotPresent
+  # -- Default image pull secret for images
+  imagePullSecrets: []
+
+image:
+  # -- Image registry for rawfile-localpv (default is global.imageRegistry)
+  registry: ""
+  # -- Image repository for rawfile-localpv
+  repository: openebs/rawfile-localpv
+  # -- Default image tag for node and controller components (uses AppVersion if empty)
+  tag: ""
+  # -- Default image pull policy for node and controller components
+  pullPolicy: IfNotPresent
 
 controller:
-  <<: *defaults
+  externalResizer:
+    image:
+      # -- Image registry for `csi-resizer`
+      registry: registry.k8s.io
```

Take from the global here too

```suggestion
      registry: ""
```